microprofile.ls/com.redhat.microprofile.ls/src/main/resources/com/redhat/microprofile/snippets/mp-metrics.json

```diff
@@ -10,7 +10,11 @@
       "\tdescription = \"${2:description}\"",
       ")"
     ],
-    "description": "An annotation that contains the metadata information when requesting a metric to be injected or produced. This annotation can be used on fields of type Meter, Timer, Counter, and Histogram. For Gauge, the @Metric annotation can only be used on producer methods/fields."
+    "description": "An annotation that contains the metadata information when requesting a metric to be injected or produced. This annotation can be used on fields of type Meter, Timer, Counter, and Histogram. For Gauge, the @Metric annotation can only be used on producer methods/fields.",
+    "context": {
+      "kind": "annotation",

```

Comment:
Could you remove kind information. For the moment I will not support it, because it will require to call JDT LS extension each time to know if completion was triggered before a class, a method, etc. I will support type for the moment which I think it is more important.